exchanges/interfaces.go

```diff
@@ -95,6 +84,10 @@ type IBotExchange interface {
 	UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error
 
 	AccountManagement
+	GetCredentials(ctx context.Context) (*Credentials, error)
+	ValidateCredentials(ctx context.Context, a asset.Item) error
```

Comment:
Just an observation: I'm not opposed to this addition as it makes it easier, but these were already exposed using GetBase()

Reply:
Yeah, that is very true, in future PR I will be updating account holdings to take in Credentials so we can segregate holdings to keys. Plus have DEX key support. I'm using this externally as well, so I thought, might as well.